src/SmartComponents/Inventory/applications/compliance/ComplianceRemediationButton.js

```diff
@@ -2,60 +2,91 @@ import React from 'react';
 import propTypes from 'prop-types';
 import { connect } from 'react-redux';
 import RemediationButton from '../../../Remediations/RemediationButton';
-import gql from 'graphql-tag';
-import { compose, graphql } from 'react-apollo';
-
-const GET_FAILED_RULES = gql`
-query FailedRulesForSystem($systemIdsQuery: String!){
-    allSystems(search: $systemIdsQuery) {
-        id,
-        rule_objects_failed {
-            ref_id,
-            title
-        }
-    }
-}
-`;
-
-const FailedRulesQuery = graphql(GET_FAILED_RULES, {
-    options: (props) => ({ variables: { systemIdsQuery: 'id ^ (' + props.selectedEntities.map(entity => entity.id) + ')' }}),
-    props: ({ data }) => (data)
-});
+import flatten from 'lodash/flatten';
 
 class ComplianceRemediationButton extends React.Component {
     constructor(props) {
         super(props);
     }
 
     /* eslint-disable camelcase */
-    formatRule = ({ title, ref_id }, system) => ({
-        id: `compliance:${ref_id}`,
+    formatRule = ({ title, ref_id }, profile, system) => ({
+        id: `ssg:rhel7|${profile}|${ref_id}`,
```

Comment:
@dLobatog this should not be hardcoded to rhel7, right?

Reply:
I am hardcoding it as I'm checking all of the inventory facts, yet none of them appear to give me any clear answer regarding the RHEL version, `os.release`, etc.. don't seem to contain that. How are you checking that? Would it be OK to default to rhel7 for the moment?

I have looked into it and I think we can infer that info from the OpenSCAP report, but I would like to get the Remediate With Ansible button to work ASAP (as marketing/CEE are trying to use it) before we fetch the OS release from the report. 

If you know of any way of getting it from the inventory, that'd be great too. Here's an example of an inventory host: 

https://gist.github.com/c8a41ce298f0a17c456b4429764644c6